type.proto

```diff
@@ -618,6 +618,8 @@ message Address {
     repeated AdministrativeRegion administrative_regions = 10;
     optional int32 house_number = 2;
     optional string label = 16;
+    // for Jormun only
+    repeated PtObject withins = 17;
```

Comment:
Do we need to put that in `Address`? Semantically, it doesn't make a lot of sense to have this information in an address. Can't we have that put in an upper object: `StopPoint` for example?

Also, `withins` doesn't mean much. I suggest we find a more explicit name about what it is.